EIPS/eip-1.md

```diff
@@ -188,7 +188,97 @@ EIPs may have a `requires` header, indicating the EIP numbers that this EIP depe
 
 ## Linking to External Resources
```

Comment:
Propose to add [ethereum/devp2p](https://github.com/ethereum/devp2p) as allowable origin